src/state/recordedState.ts

```diff
@@ -126,7 +126,7 @@ export class RecordedState {
    * The command (e.g. i, ., R, /) the user wants to run, if there is one.
    */
   public get command(): BaseCommand {
-    const list = _.filter(this.actionsRun, a => a instanceof BaseCommand);
+    const list = _.filter(this.actionsRun, a => a instanceof BaseCommand).reverse();
```

Comment:
As far as I could tell, this `get command()` isn't actually being used anywhere yet, so this should be a safe change.

However, the comment below brought me pause: 

> // TODO - disregard <Esc>, then assert this is of length 1.

If there is only supposed to be one `BaseCommand`, how come `5Y` is two commands? Not familiar with the VSCode Vim codebase, but just wanted to mention this in case it matters.

Reply:
Hmm, what are your thoughts on handling this in the processNumberedRegister call instead? Changing this seems a little bit of a workaround since the get() is currently returning the stack the way it was executed (which seems OK to me)

Not too sure on the TODO comment :/

Reply:
Well, since nobody else seems to be calling `get command()`, I guess we can define it however we wish. To me, returning the last command seems to make more sense - for example if I ran `5@q` I'd expect `.command` to refer to the macro execution, `CommandExecuteMacro` as opposed to `5`, `CommandNumber`.  Or for something like `"aD`, I'd expect `.command` to refer to the deletion `D` command, `CommandDeleteToLineEnd`, instead of the register `"a` `CommandRegister` part.

That said, you know the code better than I do, so happy to do it either way in the end! Both ways will accomplish the same purpose and get this working better which is what matters 😃

Reply:
Interesting looks like the getter for operator is reversed, lets just go with this!